Cargo.toml

```diff
@@ -13,9 +13,10 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0"
 clap = "2.33.3"
-policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", branch = "settings-options" }
-policy-fetcher = { git = "https://github.com/kubewarden/policy-fetcher", tag = "v0.1.0" }
+policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", branch = "update-dependencies" }
+policy-fetcher = { git = "https://github.com/kubewarden/policy-fetcher", branch = "refactor" }
```

Comment:
This is not intended to be merged, this PR depends on the following PR's:

- https://github.com/kubewarden/policy-evaluator/pull/2
- https://github.com/kubewarden/policy-fetcher/pull/2